users/models/identity.py

```diff
@@ -32,19 +32,71 @@
 
 class IdentityStates(StateGraph):
     """
-    There are only two states in a cycle.
     Identities sit in "updated" for up to system.identity_max_age, and then
     go back to "outdated" for refetching.
+
+    When a local identity is "edited" or "deleted", it will fanout the change to
+    all followers and transition to "updated"
     """
 
     outdated = State(try_interval=3600, force_initial=True)
     updated = State(try_interval=86400 * 7, attempt_immediately=False)
 
+    edited = State(try_interval=300, attempt_immediately=True)
+    # deleted = State(externally_progressed=True)
```

Comment:
Not sure if we want a separate delete or let it flow through as an edit that reacts to `Identity.deleted`

Reply:
Separate delete, as that way we can also use the state as a "is this user deleted" filter